### ISSUE TITLE
Jcenter needs to be removed

### ISSUE THREAD

**Issue by ivanichi** [emeraldsanto/react-native-encrypted-storage#97]
@emeraldsanto today jcenter is down, please change to mavenCentral() 

`FAILURE: Build failed with an exception.
What went wrong:
Could not determine the dependencies of task ':react-native-encrypted-storage:generateDebugRFile'.
> Could not resolve all task dependencies for configuration ':react-native-encrypted-storage:debugRuntimeClasspath'.
   > Could not resolve com.facebook.react:react-native:+.
     Required by:
         project :react-native-encrypted-storage
      > Failed to list versions for com.facebook.react:react-native.
         > Unable to load Maven meta-data from https://jcenter.bintray.com/com/facebook/react/react-native/maven-metadata.xml.
            > Could not HEAD 'https://jcenter.bintray.com/com/facebook/react/react-native/maven-metadata.xml'.
               > Read timed out
`

**Comment by mlowijs** [comment]
Same issue for me during an Azure Devops build.

**Comment by tadeas0** [comment]
Same issue here

**Comment by ismailerden** [comment]
Same issue

**Comment by dcoulter45** [comment]
Patch file if anyone needs a work around now:
 
```
diff --git a/node_modules/react-native-encrypted-storage/android/build.gradle b/node_modules/react-native-encrypted-storage/android/build.gradle
index b762894..0ef4a1c 100644
--- a/node_modules/react-native-encrypted-storage/android/build.gradle
+++ b/node_modules/react-native-encrypted-storage/android/build.gradle
@@ -1,7 +1,7 @@
 buildscript {
   repositories {
+    mavenCentral()
     google()
-    jcenter()
   }
 
   dependencies {
@@ -48,7 +48,6 @@ android {
 
 repositories {
   mavenCentral()
-  jcenter()
   google()
 
   def found = false
```

**Comment by emeraldsanto** [comment]
Closed in #98 and shipped in `4.0.3`
